### PR TITLE
Upgrade examples/guestbook to v1

### PR DESCRIPTION
**Pull request by caesarxuchao**
Update the example to v1;
Removed some default fields from the configuration files;
Update the description about load balancers and public IPs. (Reviewers: please take special care of this part as it's a quite big change)

ref #9286

@krousey @nikhiljindal @bgrant0607

### CODE REVIEW

**Comment by krousey** [comment]
@caesarxuchao I will take a look tomorrow.

**Comment by caesarxuchao** [comment]
I will update guest-go after this one has been reviewed

File: examples/guestbook/README.md [hunk 1]
```diff
@@ -146,10 +144,11 @@ redis-master            name=redis-master                         name=redis-mas
 This will cause all pods to see the redis master apparently running on <ip>:6379.  The traffic flow from slaves to masters can be described in two steps, like so.
 
 - A *redis slave* will connect to "port" on the *redis master service*
-- Traffic will be forwarded from the service "port" (on the service node) to the  *targetPort* on the pod which (a node the service listens to).
+- Traffic will be forwarded from the service "port" (on the service node) to the  *targetPort* on the pod that the service listens to.
 
 Thus, once created, the service proxy on each minion is configured to set up a proxy on the specified port (in this case port 6379).
 
+*targetPort* will default to *port* if it is omitted in the configuration. For simplicity's sake, we omit it in the following configurations.
```
Comment (krousey) [inline examples/guestbook/README.md]: You say "following configurations" here, but you omit it in the previous one as well. Maybe just don't say "following."

Comment (krousey) [inline examples/guestbook/README.md]: Nevermind, I misread.

**Pull request merged**